.gitignore

```diff
@@ -13,3 +13,5 @@ gradle/tools
 # Ignore config file generated by test
 data-prepper-main/src/test/resources/logstash-filter.yaml
 data-prepper-main/src/test/resources/logstash-conf/logstash-filter.yaml
+data-prepper-plugins/sns-sink/C:\\Work\\dlq.txt
```

Comment:
Can we just remove these files from our build? Or remove the `sns-sink` for now?

Reply:
I'll see if I can track the source of these down and disable the tests. If not, I'll just remove the plugin for now

Reply:
We don't need the plugin and it isn't working anyway. So I'm fine with just removing it. See, for example, #3448 where I removed the geoip processor.